### PR TITLE
Updates pjax to 1.9.5

### PR DESCRIPTION
**Pull request by MarcosMenegazzo**

### CODE REVIEW

File: pjax_rails.gemspec [hunk 1]
```diff
@@ -1,9 +1,9 @@
 Gem::Specification.new do |s|
   s.name    = 'pjax_rails'
-  s.version = '0.4.0'
+  s.version = '0.5.0'
   s.author  = 'David Heinemeier Hansson (PJAX by Chris Wanstrath)'
   s.email   = 'david@loudthinking.com'
-  s.summary = 'PJAX integration for Rails 3.1+'
+  s.summary = 'PJAX integration for Rails 3.2+'
```
Comment (route) [inline pjax_rails.gemspec]: Why is that?

Comment (route) [inline pjax_rails.gemspec]: Oh yeah, you're right

**Comment by route** [comment]
Thank you!

**Comment by MarcosMenegazzo** [comment]
You are welcome! Question about RubyGems: last time i checked there it was outdated.

**Comment by route** [comment]
@MarcosMenegazzo I'm going to push new version right now

**Comment by route** [comment]
Oh looks like I don't have permissions. @rafaelfranca I've sent you an email, can you give me permissions on rubygems?

**Comment by MarcosMenegazzo** [comment]
Hi @route, did you get the permissions? Rubygems still outdated

**Comment by route** [comment]
@MarcosMenegazzo sorry for the delay I'll update it soon, pretty busy with my current tasks

**Pull request merged**